src/storyblocks.js

```diff
@@ -55,11 +68,24 @@ class StoryblocksApi {
    * in JS we need to convert them back to snake case before making our
    * requests.
    *
-   * @param {object} params
+   * @param {object} object
+   * @return {object}
+   */
+  toSnakeCase (object) {
+    return mapKeys(object, (value, key) => snakeCase(key));
+  }
```

Comment:
Does having this as its own function really add much value?

Reply:
(I mean, we can do a map over the keys inline?)

Reply:
Also, is there really any point having `toSnakeCase()` and `toCamelCase()` as function on the class?  They don't use `this` at all, they're pure functions.